client/src/pages/ManageBookingPage/BookingComponent.js

```diff
@@ -0,0 +1,282 @@
+  const [reqError, setError] = useState(false);
+
+  useEffect(() => {
+    setAccepted(props.booking.accepted);
+    setDeclined(props.booking.declined);
+    setStart(moment(props.booking.start).format('YYYY-MM-DDTHH:mm'));
+    setEnd(moment(props.booking.end).format('YYYY-MM-DDTHH:mm'));
+    setModStart(moment(props.booking.start).format('YYYY-MM-DDTHH:mm'));
+    setModEnd(moment(props.booking.end).format('YYYY-MM-DDTHH:mm'));
+  }, [props]);
+
+  const fullName =
+    props.booking.sitterProfile[0].firstName + ' ' + props.booking.sitterProfile[0].lastName;
+  const handleStatus = () => {
+    if (accepted) return 'ACCEPTED';
+    else if (declined) return 'DECLINED';
+    else return 'PENDING';
+  };
+
+  const handleDate = () => {
+    const _start = moment(start);
+    const _end = moment(end);
+    const endString = () => {
+      if (_start.year() === _end.year()) {
+        if (_start.month() === _end.month()) {
+          if (_start.day() === _end.day()) return _end.format('hA');
+          else return _end.format('D MMM, hA');
+        } else return _end.format('D MMM, hA');
+      } else return _end.format('D MMM YYYY, hA');
+    };
+    return _start.format('D MMM YYYY, hA-') + endString();
+  };
+
+  const handleModalOpen = () => {
+    setModalOpen(true);
+  };
+
+  const handleModalClose = () => {
+    setModalOpen(false);
+  };
+
+  const handleMessageClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSuccess(false);
+    setError(false);
+  };
+
+  const confirmChange = async () => {
+    if (modifiedStart >= modifiedEnd) {
+      setError(true);
+      return;
+    }
+    const request = {
+      start: modifiedStart,
+      end: modifiedEnd,
+    };
+
+    await axios
+      .put(`/request/${props.booking._id}`, request)
+      .then((response) => {
+        if (!response.data.error) {
+          handleModalClose();
+          setSuccess(true);
+        }
+      })
+      .catch((error) => {
+        console.log('error: ', error);
+      });
+  };
+
+  return (
+    <Card className={classes.root} variant="outlined">
+      <CardContent className={classes.header}>
+        <Typography variant="body" className={classes.date}>
+          {handleDate()}
+        </Typography>
+        {!props.closed && (
+          <IconButton aria-label="settings" className={classes.settings} onClick={handleModalOpen}>
+            <SettingsIcon />
+          </IconButton>
+        )}
+      </CardContent>
+      <CardContent className={classes.contents}>
+        <Avatar
+          aria-label="recipe"
+          className={classes.avatar}
+          alt="T"
+          src="/images/profile_3.jpg"
+        />
+        <Typography variant="h6" className={classes.name} gutterBottom>
+          {fullName}
+        </Typography>
+        <Typography variant="h6" className={classes.status} gutterBottom>
+          {handleStatus()}
+        </Typography>
+      </CardContent>
+
+      <Modal
```

Comment:
I'm curious, any reason why you decided to use `Modal` instead of `Dialog`?

Reply:
No reason. Actually, I'm not sure differences between two of them

Reply:
Oh ok that's okay. I just think Dialog is a little easier. It's a kind of Modal so they're really both the same thing

Reply:
I will take a look about that!